src/tooltips.cpp

```diff
@@ -60,16 +60,56 @@ void tooltip::init_label()
 	rect game_canvas = video::game_canvas();
 	unsigned int border = 10;
 
+	bool mydebug=false;
```

Comment:
Yes, that's just for testing.

Reply:
Note that instead of the boolean you can use `!lg::info().dont_log(log_font)` if you're doing expensive stuff for logging. (`LOG_FT << "lsize.x,y = " << lsize.x << "," << lsize.y << ", new_text_width = " << new_text_width` is already behind this check in the macro and is not expensive anyway.)

Also IMO you should leave useful debug output in. Whoever looks at this part of the code again will presumably want similar debug output so unless the debug output is really spammy or really specific it's perfectly fine to leave it in. (Also the log improvements from #8902 could be added to make the log output easier to understand. Haven't checked if that makes sense in this PR. Just a general thought.)

Reply:
Okay.  I should probably move those to use DBG_FT anyway.   I'm just using LOG_FT here as DBG_FT logs a ton of stuff I didn't want to sift through.